### PR TITLE
FAMS3-1100

### PR DESCRIPTION
**Pull request by akroon3r**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Hasn't been tested on openshift yet, will complete a test of the working branch on openshift dev before this is approved

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## For bcgov contributors:

this PR fixes jira ticket: **FAMS3-1100**

### CODE REVIEW

File: openshift/.pipeline/Jenkinsfile [hunk 1]
```diff
@@ -10,14 +10,15 @@ def RABBITMQ = "rabbitmq"
...
+def ICBC_WEB_ADAPTER = "icbc-web-adapter"
 def DYNAMICS_ADAPTER = "dynadapter"
 def JAEGER = "jaeger"
 
 def DOMAIN = "pathfinder.gov.bc.ca"
 def SUBDOMAIN = "fams3"
 
 def GIT_URI = "https://github.com/bcgov/fams3.git"
-def GIT_REF = "master"
+def GIT_REF = "feature/icbc-web-adapter-deploy"
```
Comment (alexjoybc) [inline openshift/.pipeline/Jenkinsfile]: do you want to be master ?

File: openshift/templates/icbc-web-adapter.dc.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+                - name: JAEGER_SERVICE_NAME
+                  value: "${NAME}"
+                - name: JAEGER_SAMPLER_TYPE
+                  value: const
+                - name: JAEGER_ENDPOINT
+                  value: http://${JAEGER_COLLECTOR}:14268/api/traces
+                - name: ASPNETCORE_ENVIRONMENT
+                  value: Development
+                - name: RABBITMQ__HOST
+                  value: rabbitmq-cluster
```
Comment (alexjoybc) [inline openshift/templates/icbc-web-adapter.dc.yaml]: I see a diff here with the other ones that are using RABBIT_MQ, I guess not much impact

File: openshift/.pipeline/Jenkinsfile (hunk 1 above)
Comment (akroon3r) [reply openshift/.pipeline/Jenkinsfile]: yup I'll flip it after we've got the sign off and you've tested the web adapter. this is how tests the deployments.

Comment (akroon3r) [reply openshift/.pipeline/Jenkinsfile]: changed back to master

**Pull request merged**